apps/web/src/pages/About/Member/Member.tsx

```diff
@@ -1,33 +1,43 @@
-import Image from 'next/future/image';
+import Image from 'next/image';
```

Comment:
why did you remove future image?
bring it back and remove the wrapping div

Reply:
done